static/js/entry/style.js

```diff
@@ -26,5 +26,3 @@ import "react-telephone-input/css/default.css"
 
 import "react-draft-wysiwyg/dist/react-draft-wysiwyg.css"
 
-// This should come last to override other styles
-import "../../scss/layout.scss"
```

Comment:
no, we want to keep this - in `layout.scss` you will just need to delete the import for `user-chip.scss`

Reply:
oops, fixed @aliceriot